content/en/armory-enterprise/armory-admin/dinghy-enable.md

```diff
@@ -36,39 +36,61 @@ spec:
     config:
       armory:
         dinghy:
-          enabled: true       # Whether or not Dinghy is enabled
-          ... # Rest of config omitted for brevity
-```
+          enabled: true
+          dinghyIgnoreRegexp2Enabled: true
+{{< /prism >}}
+
+- `dinghy.enabled`: `true`; required to enable Pipelines as Code
+- `dinghy.dinghyIgnoreRegexp2Enabled`: (Optional) `true `if you want Dinghy to ignore everything other than required files.
+
+   The Regexp2 engine supports negative expressions, so you don’t need to define patterns for all files to be ignored. You can ignore everything other than required files. For example:
```

Comment:
I would like to suggest making this example more clear. Let's use something like:

For example: In your project you have some project specific files: file.js, file.ts, file.css, and files used by Dinghy: dinghyfile, minimum-wait.stage.module, maximum-wait.stage.module. If `dinghyIgnoreRegexp2Enabled` feature is enabled then you can create `.dinghyfile` with this regular expressions

```
file.(js|ts|css)
```

or

```
^(?!.*(.stage.module)|(dinghyfile)).*
```

both regular expressions bring the same result:

* file.js -> ignored
* file.ts -> ignored
* file.css -> ignored
* dinghyfile -> processed by Dinghy
* minimum-wait.stage.module -> processed by Dinghy
* maximum-wait.stage.module -> processed by Dinghy